index.bs

```diff
@@ -242,6 +229,14 @@ When the user agent is executing the <a>painting order</a>, it must populate the
 
 NOTE: A user agent might want to use a stack to efficiently compute the <a>set of owned text nodes</a> while implementing the <a>painting order</a>.
 
+Every {{Element}} has a list of <dfn>associated background image requests</dfn> which is initially an empty array.
+When the {{Element}}'s style requires a new image resource (to be displayed as background image), a new <a>image request</a> caused by the new resource is appended to the {{Element}}'s <a>associated background image requests</a>.
```

Comment:
Would be good to replace the "when..." here with an actual processing step that adds it in the relevant algorithm. I don't know if one exists though...

Reply:
I don't think CSS talks about image resources at all...